src/basic/FStar.Options.fs

```diff
@@ -766,6 +768,11 @@ let rec specs () : list<Getopt.opt> =
         "Use a previously recorded hints database for proof replay");
 
        ( noshort,
+        "use_hint_hashes",
+        ZeroArgs (fun () -> mk_bool true),
```

Comment:
I've just merged a change to master that will conflict with this line. The fix is to use `Const (mk_bool true)` instead of `ZeroArgs (mk_bool true)`

Reply:
Did you rename/remove ZeroArgs? I'll have to re-merge with master anyways, so will be fixed imminently.

Reply:
I added a layer on top of ZeroArgs / OneArg. Const gets interpreted into ZeroArgs.